network-primitives/full_node_provisioner.tf

```diff
@@ -156,7 +156,7 @@ resource "null_resource" "start-full-nodes" {
       "systemctl reenable subspace.service",
 
       # set hostname
-      "hostnamectl set-hostname ${var.network-name}-full-node-${count.index}",
```

Comment:
Same as above. Why change the hostnames of primary with domain specific details ?